new_tests/test_xmltompc80col.py

```diff
@@ -80,4 +81,4 @@ def test_trksub_9char():
         shell=True,
         check=False,
     )
-    assert result.returncode != 0 # Designed to fail...
+    assert result.returncode != 0
```

Comment:
I'm a pytest novice, I don't think this is what you want. You want a strict FAIL not an XFAIL if returncode==0, right? And you want PASS if returncode != 0.

With this commit you get XPASSED for >8 char and XFAIL for  <8 char. But in either case the overall test suite completes with no error.